docs/advanced_topics/custom_tasks.rst

```diff
@@ -0,0 +1,269 @@
+=========================
+How to add new Task types
+=========================
+
+The Workflow system introduced in Wagtail 2.9 allows users to create tasks, which represent stages of moderation.
```

Comment:
It's best not to mention specific versions in the body of the docs, as it will be an irrelevant detail in a few releases' time... if you want to mention it, you can use a `versionadded` annotation intead.

Reply:
Good point, thanks! Will remove